### ISSUE TITLE
Include person pictures

### ISSUE THREAD

**Issue by agrippine314** [meine-stadt-transparent/meine-stadt-transparent#385]
Hi :-)

The Landshut RIS contains pictures of the Stadtrat (f.ex. https://ris.landshut.de/buergerinfo/kp0051.php?__kpenr=10293). 

Would it be possible to include these in the person sections in meine-stadt-transparent? 

Might be related to #185 .

Thanks!
Annette

**Comment by stv0g** [comment]
I am currently handlings this by using a custom person.html template which directly embeds the images from our RIS in Aachen which lets us fetch the images using the OParl IDs of the respective persons.

Its not the best solution as I also need to adjust the content security policy for embedding external pictures.

I am not sure if we really need a new database model for avatars / pictures?
Maybe just a new bucket with the pictures is sufficient?
We could simply use the persons ID as the primary key for the S3 bucket..

**Comment by konstin** [comment]
>  Maybe just a new bucket with the pictures is sufficient?

Yep absolutely, plus maybe a field `has_picture` or something so we can use it in the template and avoid mass 404s

**Comment by stv0g** [comment]
@agrippine314 I am wondering how you know about the profile pictures? Are they references as a File in the Person object?

Or do you also use some well-known URLs?

@konstin I am wondering where we should download the pictures? Via an import hook? Or RIS-specific loaders?

Cheers,
Steffen

**Comment by konstin** [comment]
There's an option in oparl in principal, as far as i know no implemented it yet though so you'd need to make a small vendor specific script for that. In my experience those images normally follow a very simple schema
